index.js

```diff
@@ -16,8 +16,8 @@ var cookies = new Cookies()
 follow(process.argv[2], start)
 
 function setCookie (url, headers) {
-  var cookieHeader = headers['set-cookie']
-  if (!cookieHeader) return 0
+  if (!headers['set-cookie']) return 0
+  var cookieHeader = [].concat(headers['set-cookie'])
```

Comment:
How come you are forcing the `set-cookie` header to an array? Do the `http2` module not work the same as the core http/https modules?